source/patterns/00-atoms/doi/doi.twig

```diff
@@ -0,0 +1,11 @@
+{%- import 'atoms-html' as html -%}
+
+{%- with {doi: doi|default(null)} only -%}
```

Comment:
Move from line 3 to the end of the file (after making any further amends required). into `contextual-data.twig`, replacing the include call to this.

Then delete this file.